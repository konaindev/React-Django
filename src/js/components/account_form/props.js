export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
