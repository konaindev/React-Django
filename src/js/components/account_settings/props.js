const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
