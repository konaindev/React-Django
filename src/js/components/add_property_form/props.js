export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
