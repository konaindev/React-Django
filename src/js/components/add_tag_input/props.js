const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
