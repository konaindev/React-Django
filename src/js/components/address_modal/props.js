export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
