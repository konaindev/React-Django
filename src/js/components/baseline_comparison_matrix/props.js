import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
