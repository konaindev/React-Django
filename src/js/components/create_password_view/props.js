const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const validate = values => {
  return new Promise(res => {
    let errors = {};
    if (!values || values.length < 8) {
      errors.length = true;
    }
    setTimeout(() => res(errors));
  });
};

export const props = {
  rules,
  validate
};
