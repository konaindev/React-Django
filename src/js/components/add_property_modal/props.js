import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
