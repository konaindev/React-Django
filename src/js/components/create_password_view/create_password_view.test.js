import renderer from "react-test-renderer";
import { createStore } from "redux";
import { Provider } from "react-redux";

import CreatePasswordView from "./index";
import { props } from "./props";

jest.mock("rc-tooltip");
const _ = x => createStore(() => ({ createPassword: x }));

describe("CreatePasswordView", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={_(props)}>
          <CreatePasswordView />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
