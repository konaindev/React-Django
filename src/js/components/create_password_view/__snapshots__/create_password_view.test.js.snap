// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePasswordView renders correctly 1`] = `
<div
  className="page-auth"
>
  <div
    className="page-auth__header"
  >
    <a
      className="page-auth__back-link"
      href="/"
    >
      ← Go to Login
    </a>
    <div
      className="page-auth__logo"
    >
      <a
        href="/dashboard"
      >
        <div
          className="remarkably-logo"
        >
           
        </div>
      </a>
    </div>
  </div>
  <div
    className="page-auth__body"
  >
    <div
      className="create-password"
    >
      <div
        className="wizard-progress create-password__wizard"
      >
        <div
          className="wizard-progress__step wizard-progress__step--active"
        >
          <div
            className="wizard-progress__step-number"
          >
            1
          </div>
          Set Password
        </div>
        <div
          className="wizard-progress__step"
        >
          <div
            className="wizard-progress__step-number"
          >
            2
          </div>
          Complete Account
        </div>
      </div>
      <div
        className="create-password__body"
      >
        <div
          className="create-password__title"
        >
          Set your password
        </div>
        <div
          className="create-password__subtitle"
        >
          Enter a password to gain access to your account.
        </div>
        <form
          className="create-password__form"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="create-password__field"
          >
            <div
              className="form-field"
            >
              <div
                className="form-field__error"
              />
              <div
                className="form-field__label"
              >
                Password
              </div>
              <div
                className="form-field__input-container"
              >
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--error"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 16 16"
                  width="1em"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 8C0 3.58293 3.58293 0 8 0C12.4171 0 16 3.58293 16 8C16 12.4171 12.4171 16 8 16C3.58293 16 0 12.4171 0 8ZM11.8724 10.9435L10.9407 11.8724C10.7706 12.0425 10.4932 12.0425 10.3231 11.8724L7.99902 9.54878L5.67498 11.8724C5.50487 12.0425 5.22745 12.0425 5.05733 11.8698L4.12562 10.9435C3.95813 10.7708 3.95813 10.496 4.12562 10.3233L6.44966 7.99967L4.12824 5.67863C3.95812 5.50593 3.95812 5.22856 4.12824 5.05847L5.05733 4.12953C5.22745 3.95682 5.50748 3.95682 5.6776 4.12953L7.99902 6.45057L10.3231 4.12953C10.4932 3.95682 10.7732 3.95682 10.9407 4.12953L11.8724 5.05585C12.0425 5.22856 12.0425 5.50593 11.8724 5.67863L9.54838 7.99967L11.8724 10.3233C12.0399 10.496 12.0399 10.7708 11.8724 10.9435Z"
                    fill="currentColor"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--ok"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 16 16"
                  width="1em"
                >
                  <path
                    d="M8 0C3.58293 0 0 3.58293 0 8C0 12.4171 3.58293 16 8 16C12.4171 16 16 12.4171 16 8C16 3.58293 12.4171 0 8 0ZM12.2049 5.52439L7.91463 11.8512C7.78781 12.039 7.58537 12.1659 7.38537 12.1659C7.18537 12.1659 6.96585 12.0561 6.82439 11.9171L4.30732 9.39756C4.13659 9.22683 4.13659 8.94634 4.30732 8.77561L4.92927 8.15366C5.1 7.98293 5.38049 7.98293 5.54878 8.15366L7.1878 9.79024L10.7488 4.53659C10.8854 4.33659 11.161 4.28537 11.361 4.41951L12.0902 4.91463C12.2878 5.04878 12.3415 5.32439 12.2049 5.52439Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
          </div>
          <div
            className="create-password__field"
          >
            <div
              className="form-field"
            >
              <div
                className="form-field__error"
              />
              <div
                className="form-field__label"
              >
                Confirm Password
              </div>
              <div
                className="form-field__input-container"
              >
                <input
                  className="input input--highlight"
                  name="password2"
                  onBlur={[Function]}
                  onChange={[Function]}
                  type="password"
                />
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--error"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 16 16"
                  width="1em"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 8C0 3.58293 3.58293 0 8 0C12.4171 0 16 3.58293 16 8C16 12.4171 12.4171 16 8 16C3.58293 16 0 12.4171 0 8ZM11.8724 10.9435L10.9407 11.8724C10.7706 12.0425 10.4932 12.0425 10.3231 11.8724L7.99902 9.54878L5.67498 11.8724C5.50487 12.0425 5.22745 12.0425 5.05733 11.8698L4.12562 10.9435C3.95813 10.7708 3.95813 10.496 4.12562 10.3233L6.44966 7.99967L4.12824 5.67863C3.95812 5.50593 3.95812 5.22856 4.12824 5.05847L5.05733 4.12953C5.22745 3.95682 5.50748 3.95682 5.6776 4.12953L7.99902 6.45057L10.3231 4.12953C10.4932 3.95682 10.7732 3.95682 10.9407 4.12953L11.8724 5.05585C12.0425 5.22856 12.0425 5.50593 11.8724 5.67863L9.54838 7.99967L11.8724 10.3233C12.0399 10.496 12.0399 10.7708 11.8724 10.9435Z"
                    fill="currentColor"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--ok"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 16 16"
                  width="1em"
                >
                  <path
                    d="M8 0C3.58293 0 0 3.58293 0 8C0 12.4171 3.58293 16 8 16C12.4171 16 16 12.4171 16 8C16 3.58293 12.4171 0 8 0ZM12.2049 5.52439L7.91463 11.8512C7.78781 12.039 7.58537 12.1659 7.38537 12.1659C7.18537 12.1659 6.96585 12.0561 6.82439 11.9171L4.30732 9.39756C4.13659 9.22683 4.13659 8.94634 4.30732 8.77561L4.92927 8.15366C5.1 7.98293 5.38049 7.98293 5.54878 8.15366L7.1878 9.79024L10.7488 4.53659C10.8854 4.33659 11.161 4.28537 11.361 4.41951L12.0902 4.91463C12.2878 5.04878 12.3415 5.32439 12.2049 5.52439Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
          </div>
          <button
            className="button button--block button--uppercase button--disabled-light create-password__button"
            onClick={[Function]}
            type="submit"
          >
            Set Password
          </button>
        </form>
      </div>
    </div>
  </div>
  <div
    className="page-auth__footer"
  >
    © 2019 Remarkably. All Rights Reserved
  </div>
</div>
`;
